### PR TITLE
Bookmarks Feature Implemented Using Local Storage

### LINKED ISSUE #36: Introduce Bookmarking orgs feature
A Bookmark an org feature is to be added to all the cards, also a view bookmarks page should also be added.

### PR DESCRIPTION
**Pull request by Komachian**
Fixes #36 
- [x] Add and delete bookmark option has been added to each organization's card
- [x] A bookmarks page has been created at ```path='/bookmarks'```
- [x] The bookmarks are stored in local storage 

A [video snippet](https://user-images.githubusercontent.com/75087669/144699802-22ceb0af-b066-4bc5-bff9-f9c4c405e984.mp4) has been attached for reference
https://user-images.githubusercontent.com/75087669/144699802-22ceb0af-b066-4bc5-bff9-f9c4c405e984.mp4

### CODE REVIEW

**Comment by Sparsh1212** [comment]
@Komachian I must say commendable job. This is beyond amazing.
Since, this is quite a big PR so please allow me some time to review it properly and with some minor iterations, this will be ready to be merged 🚀🚀🚀

File: src/components/Footer.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ const Footer = () => {
                     Rishabh
                 </a>
             </h3>
+             <button className="clear-storage-btn" onClick={() => {localStorage.clear(); window.location.reload()}}>Clear Local Storage</button>
```
Comment (Sparsh1212) [inline src/components/Footer.js:32]: Please rename "Clear Local Storage" to "Reset Bookmarks"

File: src/css/mainpagecss.css [hunk 2]
```diff
@@ -21,6 +21,64 @@
...
+  bottom: 85px;
+  left: 50px;
+  background: #FFFFFF;
+  color: #000000;
+  font-weight: bolder;
+  border-radius: 10px;
+  padding: 10px;
+  cursor: pointer;
+  border: none;
+}
```
Comment (Sparsh1212) [inline src/css/mainpagecss.css:72]: Reduce border-radius to `5px` and increase the font-weight.

File: src/css/mainpagecss.css [hunk 3]
```diff
@@ -21,6 +21,64 @@
...
+  position: absolute;
+  top: 50px;
+  right: 50px;
+  background: #FFFFFF;
+  color: #000000;
+  font-weight: bolder;
+  border-radius: 10px;
+  padding: 10px;
+  cursor: pointer;
+}
```
Comment (Sparsh1212) [inline src/css/mainpagecss.css:51]: Reduce border-radius to `5px` and increase the font-weight.

File: src/css/mainpagecss.css (hunk 2 above)
Comment (Sparsh1212) [reply src/css/mainpagecss.css:72]: Ditto for Home, Bookmarks button.

File: src/components/OrganisationCard.js [hunk 4]
```diff
@@ -23,6 +22,17 @@ const OrganisationCard = (props) => {
     ],
   };
 
+  async function unBook() {
```
Comment (Sparsh1212) [inline src/components/OrganisationCard.js:25]: Rename this to `unBookmark`

**Review by Sparsh1212 — CHANGES_REQUESTED** [review]
Overall looks like a really good start!
Here are some high-level changes that must be taken care of:

- The path for `Home` should be `/` not `gsocanalyzer`. Also, every path other than `/bookmarks` must lead to `Home`. Lets say if I do `/usgugdu` then it should render the `Home` component.
- For the svgs, please create a new folder in `src` named `assets` and keep these files there. It is a good practice to separate out related files into a relevant folder together.
- It will be better to properly align the "Bookmarks" button inline with the Header. The same applies to the clear local storage button. It shall be aligned with the footer. Currently, their alignment looks a bit odd. In case you can suggest some better position for these buttons then please do.
- Both the buttons are not mobile responsive. When viewed in mobile view, the UI is looking odd. Please make them mobile responsive.

**Comment by Sparsh1212** [comment]
@Komachian Please take a look at the comments and also remove the merge conflicts.

**Comment by Komachian** [comment]
- [x] Path directing behaviour modified as requested
- [x] Icon svgs have been moved to `src/assets`
- [x] Function `unBook()` gotten rid of 
- [x] Navigation and 'Reset Bookmarks' buttons made responsive
- [x] 'Clear Storage' button renamed to 'Reset Bookmarks'
- [x] Buttons aesthetically modified as requested 
- [x] Merge Conflicts Resolved

**Comment by Komachian** [comment]
A video snippet has been attached below
https://user-images.githubusercontent.com/75087669/144976707-083dd872-3ad3-461a-8a67-ad6c1e1a48df.mp4

**Review by Sparsh1212 — CHANGES_REQUESTED** [review]
@Komachian Awesome work on the requested changes!
Just one last comment!
I can see there are 18 commits in this PR. Please make sure that it is in proper sync with master, as there are some unnecessary changes in this PR from previous commits. Let me know once it's done.

**Comment by Komachian** [comment]
The changes certainly do look in sync. The 18 commits are all the commits that were made when I first made the pull request plus every commit added by me/anyone else in the parent directory from then till now (they are listed). It seems that's because GitHub, for some reason doesn't keep track of changes to the target repository.

**Review by Sparsh1212 — APPROVED** [review]
LGTM!
Amazing work .

**Pull request merged**